Cargo.lock

```diff
@@ -1,5 +1,7 @@
 # This file is automatically @generated by Cargo.
 # It is not intended for manual editing.
+version = 3
+
```

Comment:
Are you using a newer Rust/cargo version? I didn't see this change locally.

Reply:
Yes, it was from a newer version; reverted it.